dask/dataframe/indexing.py

```diff
@@ -93,10 +93,15 @@ def __getitem__(self, key):
 
     def _loc(self, iindexer, cindexer):
         """ Helper function for the .loc accessor """
+        meta = self._make_meta(iindexer, cindexer)
         if isinstance(iindexer, Series):
             return self._loc_series(iindexer, cindexer)
         elif isinstance(iindexer, Array):
             return self._loc_array(iindexer, cindexer)
+        elif callable(iindexer):
```

Comment:
Pandas also supports callables for column indexers, would it make sense to support that as well?

Reply:
yes, I'll add it soon.